Formula/bashate.rb

```diff
@@ -0,0 +1,42 @@
+class Bashate < Formula
+  include Language::Python::Virtualenv
+
+  desc "Code style enforcement for bash programs"
+  homepage "https://github.com/openstack-dev/bashate"
+  url "https://github.com/openstack-dev/bashate/archive/2.1.1.tar.gz"
+  sha256 "a28fda309b4d7577a5d83a9f0ea8b305cb8fc626d5b320ed1bcb875ed7c576a9"
+  license "Apache-2.0"
+
+  depends_on "python@3.11"
+
+  resource "pbr" do
+    url "https://files.pythonhosted.org/packages/02/d8/acee75603f31e27c51134a858e0dea28d321770c5eedb9d1d673eb7d3817/pbr-5.11.1.tar.gz"
+    sha256 "aefc51675b0b533d56bb5fd1c8c6c0522fe31896679882e1c4c63d5e4a0fccb3"
+  end
+
+  def install
+    # This doesn't work, because of the below PBR_VERSION stuff
+    # virtualenv_install_with_resources
+
+    # Same here (expanded version of above^
+    # venv = virtualenv_create(libexec)
+    # venv.pip_install resources
+    # venv.pip_install_and_link buildpath
+
+    venv = virtualenv_create(libexec)
+    venv.pip_install resources
+    # Needed to set version for Bashate, but only after pbr has been installed
+    # otherwise we end up overriding its version as well and that's bad
+    # see https://github.com/Homebrew/homebrew-core/issues/132394
+    ENV.append("PBR_VERSION", "2.1.1")
```

Comment:
I am not quite sure if this is brew issue?

Reply:
maybe this can be `ENV["PBR_VERSION"] = version.to_s`? but this also looks a bit weird that PBR_VERSION does not match with the `pbr` dependency

Reply:
if that works, we can proceed with the merge with removing all the code comments.